section1/reflection.md

```diff
@@ -1,19 +1,31 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+I sometimes agree with the concern of being judged or seen as inexperienced. I have questions, but am sometimes afraid to ask them for fear of that judgment and how I am perceived. I know the importance of asking questions, but sometimes I try to figure the answer out and wait to see if others ask something similar. I know I need to challenge myself to not be afraid of the perception and to ask my questions.
```

Comment:
Love this! First step is just identifying it, right? 

Honestly... total newbies, beginners many times do really well at Turing because they don't come in with an ego/something to prove about their "cool coding skills". And being an educator and mom - if you can do that, you can do anything. Remember that you are a bad%$#, and asking questions is just you doing what you need to do to reach your goal of becoming a developer.